cmd/emp/destroy.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 var cmdDestroy = &Command{
-	Run:             maybeMessage(runDestroy),
+	Run:             mustConfirmAndMessageRequired(runDestroy, warningMessage),
```

Comment:
I think we can probably just compose the existing `maybeMessage` function, and prefix this chain with the confirmation. So, something like this:

```go
confirmDestroy(maybeMessage(runDestroy))
```

And then the confirmDestroy func

```go
func confirmDestroy(action func(cmd *Command, args []string)) func(cmd *Command, args []string) {
        cmd.CheckNumArgs(args)
	appname := args[0]
	warning := fmt.Sprintf("This will destroy %s and its add-ons. Please type %q to continue:", appname, appname)
	mustConfirm(warning, appname)
	action(cmd, args)
}
```

Also, note, I added the `cmd.CheckNumArgs` check, since `emp destroy` would panic of the app name wasn't provided.